docs/en_US/Tuner/HyperoptTuner.rst

```diff
@@ -1,89 +1,15 @@
-TPE, Random Search, Anneal Tuners on NNI
-========================================
-
-TPE
```

Comment:
the files for TPE and Random search are missing?

Reply:
Oh, seems I forgot to commit them.
Should not be a big problem because #4339 has included and updated them.